extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/i18n/tradfri_de.properties

```diff
@@ -0,0 +1,29 @@
+# binding
+binding.tradfri.name = TR�DFRI Erweiterung
+binding.tradfri.description = Diese Erweiterung integriert das IKEA TR�DFRI System. Durch diese k�nnen die TR�DFRI Lampen und Leuchten gesteuert werden. 
+
+# thing types
+thing-type.tradfri.gateway.label = TR�DFRI Gateway
+thing-type.tradfri.gateway.description = Steht f�r ein IKEA TR�DFRI Gateway.
```

Comment:
This should actually be fixed in the English version. The description should describe the device itself - it should not describe the thing type. So it should be something like:
```
IKEA TRÅDFRI IP Gateway
```